rpki-validator/src/main/java/net/ripe/rpki/validator3/rrdp/RrdpServiceImpl.java

```diff
@@ -261,6 +263,7 @@ private Delta readDelta(Notification notification, DeltaInfo di) {
         final byte[] deltaBody = rrdpClient.getBody(di.getUri());
         final byte[] deltaHash = Sha256.hash(deltaBody);
         if (!Arrays.equals(Hex.parse(di.getHash()), deltaHash)) {
+            rrdpMetrics.update(notification.snapshotUri, ErrorCodes.RRDP_WRONG_DELTA_HASH);
             throw new RrdpException(ErrorCodes.RRDP_WRONG_DELTA_HASH, "Hash of the delta file " + di + " is " + Hex.format(deltaHash) +
```

Comment:
These ones aren't caught at a higher level?

Reply:
Actually caught but reported as nothing alarming (fallback to snapshot), while this is seems to be something I'd like to monitor (corrupt hash).  Basically what we are reporting before this PR are metrics falling back to snapshot or new session. I need something to raise the alarm.

It is still not this one, but the IO Exception that will trigger alert.